Sources/TuistCoreTesting/Extensions/XCTestCase+Extras.swift

```diff
@@ -67,6 +57,9 @@ public extension XCTestCase {
         let decoded = XCTTry(try decoder.decode(C.self, from: json.data(using: .utf8)!), file: file, line: line)
         let encoder = JSONEncoder()
         encoder.keyEncodingStrategy = .convertToSnakeCase
+        if #available(macOS 10.13, *) {
+            encoder.outputFormatting = .sortedKeys
```

Comment:
What happens with users in older versions of macOS?

Reply:
Hm, not sure why I've added it 🤔. It's not really needed.